beacon-chain/core/helpers/committee.go

```diff
@@ -304,45 +293,41 @@ func AttestationParticipants(
 			participants = append(participants, validatorIndex)
 		}
 	}
+	sort.Slice(participants, func(i, j int) bool { return participants[i] < participants[j] })
 	return participants, nil
 }
 
-// VerifyBitfield validates a bitfield with a given committee size.
-//
-// Spec pseudocode:
+// VerifyBitfield verifies bitfield against the committee_size.
 //
-// def verify_bitfield(bitfield: bytes, committee_size: int) -> bool:
-// """
-// Verify ``bitfield`` against the ``committee_size``.
-// """
-// if len(bitfield) != (committee_size + 7) // 8:
-// return False
-//
-// # Check `bitfield` is padded with zero bits only
-// for i in range(committee_size, len(bitfield) * 8):
-// if get_bitfield_bit(bitfield, i) == 0b1:
-// return False
-//
-// return True
+// Spec pseudocode definition:
+//   def verify_bitfield(bitfield: bytes, committee_size: int) -> bool:
+//     """
+//     Verify ``bitfield`` against the ``committee_size``.
+//     """
+//     if len(bitfield) != (committee_size + 7) // 8:
+//         return False
+//     # Check `bitfield` is padded with zero bits only
+//     for i in range(committee_size, len(bitfield) * 8):
+//         if get_bitfield_bit(bitfield, i) == 0b1:
+//             return False
+//     return True
 func VerifyBitfield(bitfield []byte, committeeSize int) (bool, error) {
 	if len(bitfield) != mathutil.CeilDiv8(committeeSize) {
 		return false, fmt.Errorf(
 			"wanted participants bitfield length %d, got: %d",
-			mathutil.CeilDiv8(committeeSize),
+			(committeeSize+7)>>3,
```

Comment:
There's a few magic numbers `+7`, `>>3`, can we use variable names for them and add comments?

Reply:
removed